docs/admin_guide.md

```diff
@@ -1,6 +1,6 @@
 # Configuration
 
-This document provides detailed information about the configuration, behavior and recommended usage of `pg_diffix`.
+This reference contains descriptions of all the configuration commands, and detailed information about the configuration, behavior and recommended usage of `pg_diffix`. The [Administration Tutorial](admin_tutorial.md) contains simple step-by-step configuration instructions and several examples from real datasets, and is a good starting point.
```

Comment:
> several examples from real datasets

This is no longer the case. A link should be added to the use cases from both documents.

Reply:
@yoid2000 this line is still outdated.